### DISCUSSION TITLE
How can we increase the adoption of RedashCommunity/redash?

### DISCUSSION THREAD

**Discussion by Jengro777** [RedashCommunity/redash#30] [General]
Since the launch of the community edition, RedashCommunity/redash hasn't received much attention. We're unsure if this is due to insufficient promotion or if people are still waiting for arikfr's next move. However, as a loyal Redash fan, I hope that RedashCommunity/redash can be more widely known and that more people can participate in this project.

I believe that we should make some changes to RedashCommunity/redash, adding more powerful features that will attract more attention to the project. I have contacted some of my colleagues and we will do our best to develop certain issues.

**Comment by justinclift** [comment]
Cool.  Probably the most important thing first, is to get updated releases ready with security patches for 10.x, and also hopefully 9.x + 8.x.

That being said features and other enhancements sound like a good idea.  That's what the `master` branch is for... adding new stuff. :smile:

**Reply by Jengro777** [reply]
Getredash/redash has regained its vitality, making the positioning of RedashCommunity/redash quite awkward.

**Reply by justinclift** [reply]
Heh Heh Heh.  Nah, it's no real stress.

We still need write access to some of the other getredash repos (eg setup, website).  In theory (!) Arik will get that set up without too much delay.

If that doesn't happen though, we're going to need somewhere to work on our forks of those.

---

That being said, if the permissions and stuff all comes good in the `getredash` repos, then we can probably just archive the repos here and update the README's to point people back at the getredash ones. :smile:

**Reply by Spince** [reply]
should we archive these repos?

**Reply by justinclift** [reply]
My vote is yes, lets archive them.
